src/account_usage.rs

```diff
@@ -109,8 +109,12 @@ impl AccountUsageHandler {
     }
 
     fn handle_block_and_bank_hash(&mut self, slot: Slot) {
-        if !self.range.contains(&slot) && slot > *self.range.end() {
-            self.done = true;
+        if !self.range.contains(&slot) {
```

Comment:
fixed a bug in this while investigating...this was accumulating all packets until end of range. not only from beginning of range.